src/DynamoPackages/PackageManagerExtension.cs

```diff
@@ -49,6 +64,8 @@ public string UniqueId
         public void Dispose()
```

Comment:
@scottmitchell as a sanity check - it might be good to make sure this dispose method gets called during a test.

Reply:
Yes, it appears this does get called during serialization tests